marketplace_builder/views/pages/tutorials/performance/cache.liquid

```diff
@@ -0,0 +1,111 @@
+---
+converter: markdown
+metadata:
+  title: Cache 
+  description: Detailed descriptions of page, action, and fragment caching with examples.  
+slug: tutorials/performance/cache
+searchable: true
+---
+
+## Page caching
+
+Page caching means you create a static HTML site, and serve it bypassing the whole server side, including authorization. It's super fast, but you can rarely use it. For example, it's great to use this for the home page, to make it very fast for non-authenticated users, but you can't do the same for a dashboard, because the user has to be authenticated. 
+
+**Example:** 
+
+#### views/pages/home.liquid: 
+
+```liquid
+{% raw %}
+---
+slug: /
+static_cache:
+  expire: 10 # static html page will be cached for 10 seconds
+---
+... 
+{% endraw %}
+```
+
+{% include 'alert/note', content: 'In the old infrastructure of PlatformOS, you used `cache_for`. See the example above for the new infrastructure.' %}
```

Comment:
do we really want to mention that?

Reply:
@Slashek Please advise! Thank you!

Reply:
This is the only comment keeping this topic from merging. The information is correct, so let's publish it like this and remove it later if necessary.